public/about.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+<title>about me</title>
+
+<meta name="Sally Moore - about me" content="Web developer with a Ph.D. in clinical psychology. Likes puppies." charset="UTF-8">
```

Comment:
I think you might have been looking for the the `description` meta tag. Every meta tag is distince; the `charset` reference you included here is very important, but I'm not sure it would work as intended in the current arrangement. It should look more like `<meta charset='utf-8'>`.